wemake_python_styleguide/visitors/ast/complexity/function.py

```diff
@@ -51,11 +65,11 @@ def __init__(self) -> None:
         self.awaits: _FunctionCounter = defaultdict(int)  # noqa: WPS204
```

Comment:
Let's refactor this part as well! What classes do you like to extract from these attributes?